spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/KubernetesConfigDataLocationResolver.java

```diff
@@ -130,7 +130,7 @@ protected final boolean isRetryEnabledForSecrets(SecretsConfigProperties secrets
 				&& secretsProperties.failFast();
 	}
 
-	protected KubernetesNamespaceProvider kubernetesNamespaceProvider(Environment environment) {
```

Comment:
this sits in commons package as `protected`, but implementations: `KubernetesClientConfigDataLocationResolver` and `Fabric8ConfigDataLocationResolver` don't override them in any meaningful way. Most probably just a left over of the initial though process when implementing.